src/services/VersionValidatorService.php

```diff
@@ -58,7 +58,7 @@ public function getActualVersionData()
             ));
 
             $releaseJsonData = Tools::file_get_contents(self::PLUGIN_RELEASE_DATA_URL, false, $context);
-            $releaseData = Tools::jsonDecode($releaseJsonData, true);
+            $releaseData = json_decode($releaseJsonData, true);
```

Comment:
Isn't it required to use the `Tools` class anymore?

Reply:
I don't know php at all, but all I know is that using `Tools::jsonDecode` does not work for me on prestashop 8 and php 8.1, whereas `json_decode` appears to not cause internal server errors for prestashop.

I don't know how you test the plugin currently, but maybe you should try out the binami docker image?

Please see:
https://github.com/bitnami/containers/tree/main/bitnami/prestashop#how-to-use-this-image

Reply:
@rkistaps It appears it was removed in prestashop here
https://github.com/PrestaShop/PrestaShop/commit/c1d84724067c6fecf75745b9f7eb8d419fe9002d

Reply:
I just commented on the version verification and works well